sources/packages/web/src/router/AESTRoutes.ts

```diff
@@ -303,6 +303,15 @@ export const aestRoutes: Array<RouteRecordRaw> = [
               clientType: ClientIdType.AEST,
             },
           },
+          {
+            path: AppRoutes.ApplicationVersionDetail,
+            name: AESTRoutesConst.APPLICATION_VERSION_DETAILS,
+            props: true,
+            component: StudentApplicationView,
```

Comment:
Now that we send the `versionid` we have to make a minor adjustment in `StudentApplicationView.vue` otherwise the API call to get application data will always use the `applicationId` and not the `versionId`.

Issue:

![image](https://github.com/user-attachments/assets/ca51e4a2-01a3-480a-9731-fe082b55486b)

My best suggestion on solution:
![image](https://github.com/user-attachments/assets/6e4128cf-a4cf-4a3a-9e12-f9b107946b61)